docs/static/configuration.asciidoc

```diff
@@ -663,8 +667,39 @@ output {
 }
 ----------------------------------
 
+[discrete]
+[[date-math]]
+===== Date math in conditionals
+
+Date math in conditionals is not currently supported, but a workaround is available. 
+Put the date calculation in a field so that you can use the field reference in a conditional. 
+
+*Example* 
+
+This statement won't work: 
```

Comment:
I think a little more context would help here:

Something like:

This attempt to use add a field depending on the ingestion time won't work using `sprintf time format` directly